src/Hermes/SendEmailHandler.php

```diff
@@ -49,7 +49,7 @@ public function handle(MessageInterface $message): bool
             $payload['mail_template_code'],
             $payload['params'] ?? [],
             $payload['context'] ?? null,
-            $attachments,
+            $attachments ?: [],
```

Comment:
Nice catch. This second failsafe is probably not necessary, since `$attachments` will exist as array because of line 40. I'll amend that and merge to the master. Thanks for the fix!